ui/src/notebooks/components/header/Submit.tsx

```diff
@@ -16,6 +17,8 @@ import {RemoteDataState} from 'src/types'
 const PREVIOUS_REGEXP = /__PREVIOUS_RESULT__/g
 const COMMENT_REMOVER = /(\/\*([\s\S]*?)\*\/)|(\/\/(.*)$)/gm
 
+const fakeNotify = notify
```

Comment:
what does this do?

Reply:
It fakes a notify up the middle, qb drops back, waits, slings it downfield. TOUCHDOWN!